Backend/dev/migrations/dynamic-offer-driver-app/0485-added-pn-and-payout-vpa-status.sql

```diff
@@ -1,9 +1,10 @@
 UPDATE atlas_driver_offer_bpp.driver_information SET payout_vpa_status = 'VIA_WEBHOOK' where payout_vpa is not null;
 
 INSERT INTO atlas_driver_offer_bpp.merchant_push_notification (
-    fcm_notification_type, key, merchant_id, merchant_operating_city_id, title, body, language, created_at, updated_at
+    id, fcm_notification_type, key, merchant_id, merchant_operating_city_id, title, body, language, created_at, updated_at
 )
 SELECT
+    atlas_driver_offer_bpp.uuid_generate_v4(),
```

Comment:
We can avoid manually changing existing queries. Should be handled with default condition while adding `id`.
```
ALTER TABLE atlas_driver_offer_bpp.merchant_push_notification ADD COLUMN id UUID DEFAULT md5(random()::text || clock_timestamp()::text)::uuid;
```